### DISCUSSION TITLE
Standalone 2.3.100 Error updating to 2.3.110

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#7486] [Unsupported Versions] [answered]
Error when updating standalone from 2.3.100 to 2.3.110. 

There is an issue rendering the manager's pillars. Please correct the issues in the sls files mentioned below before running SOUP again.

"Rendering SLS 'global' failed. Please see master log for details."

![image](https://user-images.githubusercontent.com/70167373/157535281-f09da205-a4f8-4256-b41d-f4b7c7cd4c45.png)

![image](https://user-images.githubusercontent.com/70167373/157537246-2fba8876-e0f0-45ed-82de-ad366adb3b5f.png)
![image](https://user-images.githubusercontent.com/70167373/157537281-7490d436-a9b9-4ca7-9086-f3b9dbca27f0.png)

Fixed it. This was the issue. After taking the second subnet '192.168.2.0/24' out and re-running soup everything worked.
![image](https://user-images.githubusercontent.com/70167373/157538398-b159e340-6105-4bfa-8e93-53f7413a0f3c.png)

**Comment by dougburks** [comment, marked as answer]
Yep, that would be a syntax error. You can specify multiple networks for `hnmanager` like this:
```
hnmanager: '10.0.0.0/8,192.168.0.0/16,172.16.0.0/12'
```
